pkg/package.go

```diff
@@ -161,3 +161,36 @@ func (p Package) MarshalText() string {
 func (p Package) String() string {
 	return fmt.Sprintf("package '%s'", p.PID.String())
 }
+
+func (p Package) Create(rootPaths []string, filterFile string) (string, error) {
```

Comment:
keep consistent naming, previously `filter` is here named as `filterFile` right?